### ISSUE TITLE
max-len auto fix, is it possible?

### ISSUE THREAD

**Issue by mttzzz** [antfu/eslint-config#582]
### Clear and concise description of the problem

While ESLint's max-len rule can enforce line length, it doesn't automatically break long lines into shorter ones. This is different from tools like Prettier, which can automatically format code to meet such length requirements.  is there a recommended approach to achieve this with the current setup?

### Suggested solution

auto fix

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by antfu** [comment]
- https://github.com/eslint-stylistic/eslint-stylistic/issues/73
- https://discord.com/channels/1168956330468450334/1168956969881706630/1225950262422802574

TL;DR: No, not yet, and it's not progressing any sooner. But open for contributions.
